docs/vivisect/quickstart.rst

```diff
@@ -22,7 +22,7 @@ Packages for vivisect are built and push to the Python Package index at https://
 
 Which will get you the latest vivisect for use in headless mode. If would also like a cool hacker UI to do your reverse engineering in, you can instead run::
 
-    pip install vivisect[gui]
+    pip install "vivisect[gui]"
 
 Which will also install the PyQt5 dependecies necessary for running the vivisect UI.
 
```

Comment:
should we add in the Windows way of installing PyQt5 and PyQtWebEngine here?

pip install pyqt5
pip install pyqtwebengine

Reply:
nm.  this is what `vivisect[gui]` for